wire-runtime/src/jvmMain/kotlin/com/squareup/wire/ProtoAdapter.kt

```diff
@@ -41,7 +42,14 @@ actual abstract class ProtoAdapter<E> actual constructor(
     else -> commonCreateRepeated()
   }
 
-  constructor(fieldEncoding: FieldEncoding, type: Class<*>): this(fieldEncoding, type.kotlin)
+  // Obsolete; for Java classes generated before typeUrl was added.
```

Comment:
Use `@Deprecated`?

Reply:
Dixit Jesse: "A user seeing a deprecated method in the generated would be too sad."

Reply:
Yeah, users can’t fix the deprecation warnings so it’s just needlessly adding warnings to users’ builds